e2e/src/usecases/HandleDeepLinkSend.js

```diff
@@ -16,13 +18,34 @@ const launchDeepLink = async (url, newInstance = true) => {
   await launchApp({ url: url, newInstance: newInstance })
 }
 
+const launchClevertapDeepLink = async (url) => {
+  await device.terminateApp()
+  await launchApp({
+    userNotification: {
+      trigger: {
+        type: DetoxConstants.userNotificationTriggers.push,
+      },
+      payload: {
+        wzrk_dl: url,
+      },
+    },
+  })
+}
+
 const openDeepLink = async (payUrl) => {
   await reloadReactNative()
   await device.openURL({ url: payUrl })
 }
 
 export default HandleDeepLinkSend = () => {
   describe('When Launching Deeplink - App Closed', () => {
+    it('The should handle deeplink to another screen', async () => {
```

Comment:
Looks like we could move this to [handleNotifications](https://github.com/valora-inc/wallet/blob/f3b008bb6297ec679419ad8604449e474029632a/e2e/src/usecases/HandleNotification.js) instead: 
```suggestion
    it('Launch app from push notification should allow deeplink to another screen', async () => {
```